server/src/test/java/org/elasticsearch/search/profile/query/QueryProfileShardResultTests.java

```diff
@@ -33,7 +33,9 @@ public static QueryProfileShardResult createTestItem() {
         if (randomBoolean()) {
             rewriteTime = rewriteTime % 1000; // make sure to often test this with small values too
         }
-        return new QueryProfileShardResult(queryProfileResults, rewriteTime, profileCollector);
+
+        long vectorOperationsCount = randomNonNegativeLong();
+        return new QueryProfileShardResult(queryProfileResults, rewriteTime, profileCollector, vectorOperationsCount);
```

Comment:
Its also possible for it to not be provided, we need the null case.

Reply:
Added